internal/common/customplanmodifier/create_only_default_bool.go

```diff
@@ -0,0 +1,69 @@
+package customplanmodifier
```

Comment:
moving plan modifier that supports default value in separate file, no modifications made to the implementation